packages/ListBoxBrowser/src/components/CustomListBox/CustomListBox.js

```diff
@@ -57,7 +57,7 @@ export default function CustomListBox(props) {
 
   return (
     <ListBox key={id} height={height} isMulti={isMulti} isInline onChange={onChange}>
-      <ListBox.Trigger isHidden></ListBox.Trigger>
+      <ListBox.Trigger isHidden />
```

Comment:
Not sure why `eslint` started complaining about this, but I guess it counts as a `PATCH` update 🙄